docs/hardware.rst

```diff
@@ -427,28 +449,19 @@ Intel 7th-gen NUC
 We have tested and can recommend the `NUC7i5BNH <https://ark.intel.com/content/www/us/en/ark/products/95067/intel-nuc-kit-nuc7i5bnh.html>`__.
 
 The NUC7i5BNH has soldered-on wireless components, which cannot easily be
-removed. For security reasons, we recommend that you take the following steps
-to disable wireless functionality:
-
-- before installation of the RAM and storage, disconnect the wireless antennae
-  leads.
+removed. Before installation of the RAM and storage, we recommend that you
+disconnect the wireless antennae leads:
 
 |NUC7 leads|
 
-- before the initial OS installation, boot into the BIOS by pressing **F2** at
-  startup, navigate to **Advanced > Devices > Onboard Devices**, and disable
-  unwanted hardware - everything except **LAN**.
-
-|Visual Bios|
+Before the initial OS installation, boot into the BIOS by pressing **F2** at
+startup and adjust the system configuration:
 
+- Under **Advanced ▸ Devices ▸ Onboard Devices**, disable all onboard devices
+  other than LAN: audio, audio DSP, microphone, Thunderbolt, WLAN, Bluetooth,
+  and SD card.
 
 .. |NUC7 leads| image:: images/hardware/nuc7-leads.jpg
-.. |Visual BIOS| image:: images/hardware/visualbios.png
-
-Other 7th-generation NUCs have also been reported to work, although we have not
-tested them. For example, the `NUC7i5DNHE <https://www.intel.com/content/www/us/en/products/boards-kits/nuc/kits/nuc7i5dnhe.html>`__ uses the same Ethernet chipset as the NUC7i5BNH,
-and also has a removable wireless card, simplifying the server setup process.
-However, it may be harder to find a retail source for this model.
```

Comment:
Given that the NUC7 was discontinued last year and we do not recommend installing on NUC7s for new installations at this point, I figured it was best to remove this outdated guidance.